src/components/admin/donations/grid/Grid.tsx

```diff
@@ -166,6 +182,14 @@ export default observer(function Grid() {
                 <RestoreIcon />
               </IconButton>
             </Tooltip>
+            <Tooltip title={t('donations:invalidate.icon')}>
```

Comment:
Invalidating donation should be outside of `params.row?.provider === PaymentProvider.stripe` condition, as we might need to invalidate bank donations as well.

Reply:
Fixed.